live/src/main/java/org/dbpedia/extraction/live/queue/LiveQueue.java

```diff
@@ -105,6 +105,28 @@ public static String getPriorityDate(LiveQueuePriority priority){
         return "";
     }
 
+    /*
+    * Used to get a list of the next items in the queue.
+    *
+    * To achieve this we have to remove them from the queue one by one, save the information we need,
+    * and add them again to the queue.
+    * Although this isn't ideal, the PriorityBlockingQueue doesn't give us other viable options.
+    * */
+    public static ArrayList<String> getNextQueuedItems(){
```

Comment:
removing and re-inserting is not a good idea here
I'd suggest to use `.peek()' for one item or `.iterator()` for multiple items
http://docs.oracle.com/javase/7/docs/api/java/util/concurrent/PriorityBlockingQueue.html